pkg/util/errors/wrapper_test.go

```diff
@@ -0,0 +1,23 @@
+package errors
+
+import (
+	"errors"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("wrapper.go", func() {
+	Context("when Wrap() called with proper error and message", func() {
+		BeforeEach(func() {
+		})
+		It("should return a formatted error", func() {
+			sut := Wrap(errors.New("dummy context"), "testing")
+			Expect(sut.Error()).NotTo(BeNil())
+			Expect(strings.Contains(sut.Error(), "gcp-project-operator/pkg/util/errors/wrapper_test.go")).Should(BeTrue())
```

Comment:
change to https://onsi.github.io/gomega/#containsubstringsubstr-string-args-interface same as a previous comment

Reply:
agreed as well